docs/SPEC.md

```diff
@@ -2146,6 +2146,69 @@ Examples:
     | 0002  | "temp" | 75 | "temp" | 56 |
     | 0003  | "temp" | 72 | "temp: | 55 |
 
+#### Union
+
+Union concatenates two input streams into a single output stream.  In tables that have identical
+schema and group keys, contents of the tables will be concatenated in the output stream.
+
+Example:
+
+* SF_Weather
+* Group Key for both tables {"_field"}
+
+    | _time | _field | _value |
+    | ----- | ------ | ------ |
+    | 0001  | "temp" | 70 |
+    | 0002  | "temp" | 75 |
+
+    | _time | _field | _value |
+    | ----- | ------ | ------ |
+    | 0001  | "humidity" | 81 |
+    | 0002  | "humidity" | 82 |
+
+* NY_Temperature
+* Group Key for both tables {"_field"}
+
+    | _time | _field | _value |
+    | ----- | ------ | ------ |
+    | 0001  | "temp" | 55 |
+    | 0002  | "temp" | 56 |
+
+    | _time | _field | _value |
+    | ----- | ------ | ------ |
+    | 0001  | "pressure" | 29.82 |
+    | 0002  | "pressure" | 30.01 |
+
+`union(tables: {sf: SF_Temperature, ny: NY_Temperature})` produces:
```

Comment:
@jlapacik   what happens if there's > 2 tables for join?   

@wolffcm  do we want to support N >= 2 tables for union?  Seems like it wouldn't be much more complex than the N=2 case.

Reply:
@aanthony1243 join does not support joining more that two streams at the moment.

Reply:
It seems like we could handle more than 2 streams with no change at all to the processing algorithm.  Is there any reason not to support an arbitrary number of of tables?

Reply:
I don't think so. I would say do it if it's trivial.

Reply:
@jlapacik probably not urgent, but the spec leaves it undefined what happens if you pass >2 tables to join

Reply:
@aanthony1243 there are also a few other things I need to change in the join spec. I'll put up a follow-up PR.

Reply:
>  Should it use a simple list of streams instead?

I was thinking that union should just take a list of tables.